bin/install_geoserver.sh

```diff
@@ -291,6 +292,10 @@ chgrp users /usr/local/share/geoserver
 chmod g+w /usr/local/share/geoserver
 cp /usr/local/share/gisvm/app-data/geoserver/ne_10m*.sld \
    /usr/local/share/geoserver/
+# sample data for NetCDF section of quickstart (from GeoServer 2.10.4 as removed in later versions)
+wget --progress=dot:mega \
+  -O /usr/local/share/geoserver/polyphemus_20120401.nc \
+  "https://github.com/geoserver/geoserver/raw/2.10.4/doc/en/user/source/rest/examples/polyphemus_20120401.nc"
```

Comment:
Please add the new dataset to the common data folder and link it to the desired path if needed.